src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -86,7 +88,9 @@ Author: Daniel Kroening, kroening@kroening.com
   "(undefined-function-is-assume-false)" \
   "(remove-function-body):"\
   "(splice-call):" \
-  OPT_REMOVE_CALLS_NO_BODY
+  OPT_REMOVE_CALLS_NO_BODY \
+  OPT_REPLACE_FUNCTION_BODY
```

Comment:
I think this might not be worth renaming to `OPT_GENERATE...`?

Reply:
done